src/dispatch/signal/scheduled.py

```diff
@@ -49,3 +49,25 @@ def consume_signals(db_session: SessionLocal, project: Project):
 
         if signal_instances:
             plugin.instance.delete()
+
+
+@scheduler.add(every(1).minutes, name="signal-process")
+@scheduled_project_task
+def process_signals(db_session: SessionLocal, project: Project):
+    """Processes signals and create cases if appropriate."""
+    signal_instances = (
```

Comment:
should we constrain it more somehow? i think this will grow forever and potentially get slow

Reply:
Anytime a signal is processed it should now have a value for `filter_action`. So only unprocessed signals should get returned here.